test/BUILD

```diff
@@ -24,6 +24,33 @@ haskell_test(
     ],
 )
 
+haskell_test(
+    name = "third-party-test",
+    srcs = ["third-party-test.hs"],
+    deps = [
+        hazel_library("aeson"),
+        hazel_library("aeson-extra"),
+        hazel_library("cassava"),
+        hazel_library("conduit"),
+        hazel_library("entropy"),
+        hazel_library("fuzzyset"),
+        hazel_library("hsndfile"),
+        hazel_library("htaglib"),
+        hazel_library("http-client"),
+        hazel_library("language-c"),
+        hazel_library("lens"),
+        hazel_library("network"),
+        hazel_library("postgresql-libpq"),
+        hazel_library("pretty-show"),
+        hazel_library("text-metrics"),
+        hazel_library("unix-compat"),
+        hazel_library("unix-time"),
+        hazel_library("vector"),
+        hazel_library("wai-app-static"),
```

Comment:
Note that wai-app-static doesn't work anymore due to the mangling

Reply:
Have pushed a quick fix for this here if you need it today https://github.com/FormationAI/wai/commit/9217512fae1d6c2317447b257f478005efb55ef7

Reply:
Huh, how stable are those suffixes? I was worried changing the git commit would cause a new hash in the file to be suffixed and you'd have a circular update problem.

Reply:
ohhh ok that less scary than I thought at first glance. Yeah Im hoping there's a way to make the TH embed work, similar to #60, but I haven't fully grokked whats going on here.